.github/workflows/ci.yml

```
@@ -0,0 +1,58 @@
+name: CI
+on:
+  push:
+    branches:
+      - main
+      - release/**
+  pull_request:
+
+jobs:
+  linting:
+    name: "Run linters"
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout code
+      - uses: actions/setup-python@v2
```

```suggestion
      - uses: actions/checkout@v4
        name: Checkout code
      - uses: actions/setup-python@v5
```
nit: we should bump these to match the `build` check: https://github.com/getsentry/infra-event-notifier/blob/main/.github/workflows/build.yml#L15-L16